### ISSUE TITLE
kerning pair μη

### ISSUE THREAD

**Issue by ousia** [dbenjaminmiller/garamond-libre#3]
@dbenjaminmiller,

I have the impression that the pair μη could have a bit more tight kerning:

[![glibre-000001](https://user-images.githubusercontent.com/203767/79161817-6ccef100-7ddc-11ea-84a9-0d7cf1426348.png)](https://github.com/dbenjaminmiller/garamond-libre/files/4471893/glibre.pdf)

What do you think? (I came to this idea by using it to compose a text with some Greek words in it.)

**Comment by dbenjaminmiller** [comment]
Yeah, that's true. I'd want to look to see if it's a problem with the letterspacing of μ generally or with the kerning pair only.

**Comment by ousia** [comment]
Another sample of μ in context:

[![glibre-000001](https://user-images.githubusercontent.com/203767/80917322-545c5180-8d5e-11ea-8897-871455351f32.png)](https://github.com/dbenjaminmiller/garamond-libre/files/4570801/glibre.pdf)
